### ISSUE TITLE
Key press balance for sliders

### ISSUE THREAD

**Issue by Matthew-phet** [phetsims/molarity#128]
**Solute Amount. slider**
-  key press at "no"					(1)
-  key presses at "a little"			(3)
-  key presses at "a low amount of"      	(3)
-  key presses at "some"				(6)
-  key presses at "a bunch of"			(3)
-  key presses at "a lot of"			(3)
-  key press at "max amount of"	        (1) 

**Solution Volume slider**

-  key press at “full”				 (1)
-  key presses at “nearly full”		 (3)
-  key presses at “over half full”	 (3)
-  key presses at “half full”		 (6)	
-  key presses at “under half full”	 (3)
-  key presses at “nearly empty”	 (3)
-  key press “at lowest amount” 	 (1)

**Comment by terracoda** [comment]
@Matthew-phet, I am going to clarify the above and change what you posted. I made an error in the agenda for the number of key presses for the solute slider (sorry).

The Solute Amount slider has 20 default steps (i.e., 21 default positions) at the default step-size of 0.050 moles.

The Solution Volume slider has 16 default steps (i.e., 17 default positions) at the default step-size of 0.050 liters. 

The ranges that were implemented seemed to create an unbalanced stepping experience in terms of the number of default steps (key presses) for each range. 

@twant, the ranges that were implemented seemed to create an unbalanced stepping experience in terms of the number of default steps (key presses) for each range. What we would like you to do is to even out ranges so that each has the following number of key presses. 

I think these ranges will result in the desired number of default steps. The Solution Volume slider may still seem lop-sided (more steps in upper half of beaker), but that's because the volume doesn't go to zero litres.

### Solute Amount slider
**Solute Amount steps and ranges**
-  "no" - **1 step** (min position) at value: 0.000 moles 
-  "a little" - **3 default steps** at values: 0.050, 0.100, 0.150; full range: 0.001 - 0.150 moles; range: 0.001-0.150 moles
-  key presses at "a low amount of" - **4 default steps** at values: 0.200, 0.250, 0.300, 0.350; range: 0.151-0.350 moles
-  key presses at "some" - **5 default steps** at values: 0.400, 0.450, 0.500, 0.550, 0.600; range: 0.351-0.600 moles
-  key presses at "a bunch of" - **4 default steps** at values: 0.650, 0.700, 0.750, 0.800; range: 0.601-0.800 moles
-  key presses at "a lot of" - **3 default steps** at values: 0.850, 0.900, 0.950; range: 0.850-0.999 moles
-  key press at "max amount of" - **1 step** (max position) at value: 1.000 moles  

### Solution Volume slider
**Solution Volume steps and ranges**

-  “full”  - **1 step** (max position) at value: 1.000 liters; full range: 0.001 litres 
- “nearly full” - **4 default steps** at values: 0.800, 0.850, 0.900, 0.950; full range 0.800 - 0.999 litres
-  “over half full” - **5 default steps** at values: 0.550, 0.600, 0.0650, 0.700, 0.750; full range 0.550 - 0.799 litres
-  “half full” - **1 step** at value: 0.500 litres; full range: 0.001 litres	
-  “under half full” - **3 default steps** at values: 0.350, 0.400, 0.450; full range: 0.350 - 0.499
-  “nearly empty” - **2 default steps** at values: 0.250, 0.300; full range: 0.002 - 0.349
-  “at lowest amount” - **1 step** (min position) at value: 0.200 litres; full range: 0.001 litres

@Matthew-phet and @twant, does this seem more logical now? I'm leaving assigned to @twant, but please re-assign to me or @Matthew-phet if something is missing.

**Comment by terracoda** [comment]
@twant, the key presses are good. Closing.
